apps/lobe-chat/config.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "Lobe Chat",
+  "available": true,
+  "port": 3210,
+  "exposable": true,
+  "id": "lobe-chat",
+  "description": "LobeChat is an open-source, high-performance chatbot framework that supports speech synthesis, multimodal, and extensible (Function Call) plugin system.",
+  "tipi_version": 1,
+  "version": "0.116.4",
+  "categories": ["utilities", "ai", "development"],
+  "short_desc": "LLM chatbot framework",
+  "author": "https://github.com/lobehub",
+  "source": "https://github.com/lobehub/lobe-chat",
+  "website": "https://github.com/lobehub/lobe-chat",
+  "form_fields": [
+    {
+        "type": "text",
+        "label": "OpenAI API Key",
+        "required": true,
+        "env_variable": "OPENAI_API_KEY"
+    },
+    {
+        "type": "fqdn",
```

This should be an URL
```suggestion
        "type": "url",
```